pep-0675.rst

```diff
@@ -562,8 +562,23 @@ annotating it as ``x: LiteralString``:
 Runtime Behavior
 ================
 
-We propose an implementation for ``typing.LiteralString`` similar to that for
-``typing.Self`` from :pep:`673`.
+We propose an implementation for ``typing.LiteralString`` similar to
+``typing.NoReturn``:
+
+::
+
+    @_SpecialForm
```

Comment:
I'd rather not put this in a PEP as it's an implementation detail. I think you can just say that it will behave like NoReturn at runtime and leave it at that.